model/SearchEngine/Driver/Elasticsearch/ElasticSearchIndexer.php

```diff
@@ -194,8 +194,8 @@ public function searchResourceByIds(array $ids): array
         $hits = $response['hits'] ?? [];
 
         $document = [];
-        if ($hits && isset($hits['hits'], $hits['total'], $hits['total'])) {
-            $document = current($hits['hits']);
+        if ($hits && isset($hits['hits'], $hits['total'])) {
+            $document = current($hits['hits']) ?: [];
```

Comment:
How can `hits` be set, but not an array?

Reply:
> If the internal pointer points beyond the end of the elements list or the array is empty, current() returns false.

Seems that value under `hits` index may be an empty array, while `isset` checks only that key exists.